go.mod

```diff
@@ -1,17 +1,18 @@
 module github.com/parca-dev/parca-load
 
-go 1.18
+go 1.19
 
 require (
-	github.com/bufbuild/connect-go v1.3.0
+	buf.build/gen/go/parca-dev/parca/connectrpc/go v1.11.0-20230823175648-8e6b94c22fce.1
+	buf.build/gen/go/parca-dev/parca/protocolbuffers/go v1.31.0-20230823175648-8e6b94c22fce.1
```

Comment:
The versioning here is very a pseudo Go module version that captures a "synthetic version" comprising of:

- plugin version (major.minor.patch). In this case its `v1.31.0` of protocolbuffers/go, https://buf.build/protocolbuffers/go
- module version (timestamp + short module commit), `20230823175648-8e6b94c22fce`, https://buf.build/parca-dev/parca/docs/8e6b94c22fce471a9f4f67542bbb7db8
- `.1` is the revision of the plugin (usually 1 in case we messed up, since the plugin version is controlled by a source we don't control)

More details on versioning here, https://buf.build/docs/bsr/remote-packages/go#versions

Lastly, the source/builds of the plugins is now open-source, https://github.com/bufbuild/plugins